urdf2webots/parserURDF.py

```diff
@@ -944,6 +986,12 @@ def parseGazeboElement(element, parentLink, linkList):
                     if link.name == name:
                         link.forceSensor = True
                         break
+        elif plugin.hasAttribute('filename') and plugin.getAttribute('filename').startswith('libgazebo_ros_p3d'):
+            p3d = P3D()
```

Comment:
Maybe a warning here to notify a user that parameters `xyzOffsets` and `rpyOffsets` cannot be handled

Reply:
Yes, that is something I'm also concerned about. I guess you could also add a Transform node to handle this, but I got only little time to work on these changes. Therefore, I would suggest to add a kind of "not implemented yet" warning?

Reply:
A warning should be enough. If the `xyzOffsets` or `rpyOffsets` parameter is present we should just print a warning.

Reply:
The latest commit includes the suggested changes. A warning is now thrown when either `xyzOffsets` or `rpyOffsets` parameter is present.